### PR TITLE
Fixed: Speed up load and RSS

### PR DESCRIPTION
**Pull request by ta264**
#### Database Migration
YES (185, add CleanTitle indices to Alternative Titles and Translations)

#### Description
- Fix language mapping
- Grab mediacover fileinfos in one go instead of one at a time
- Get the movies and other components of the response in parallel
- Fix FindByTitle so it doesn't repeat queries to DB
- Fix FindByTitle so it uses the index instead of a table scan

#### Todos
- [ ] Tests
- [x] Translation Keys

#### Issues Fixed or Closed by this PR

* #

### CODE REVIEW

**Comment by Qstick** [comment]
Serializing seems to busted for custom filters, when adding one or having one in db:

```
2020-10-7 21:07:24.0|Error|Microsoft.AspNetCore.Server.Kestrel|Connection id "0HM3B4PV6P27M", Request id "0HM3B4PV6P27M:00000001": An unhandled exception was thrown by the application.

[v3.0.0.3883] System.NotSupportedException: The collection type 'Newtonsoft.Json.Linq.JObject' is not supported.
   at System.Text.Json.JsonPropertyInfoNotNullable`4.GetDictionaryKeyAndValueFromGenericDictionary(WriteStackFrame& writeStackFrame, String& key, Object& value)
   at System.Text.Json.JsonPropertyInfo.GetDictionaryKeyAndValue(WriteStackFrame& writeStackFrame, String& key, Object& value)
   at System.Text.Json.JsonSerializer.HandleDictionary(JsonClassInfo elementClassInfo, JsonSerializerOptions options, Utf8JsonWriter writer, WriteStack& state)
   at System.Text.Json.JsonSerializer.Write(Utf8JsonWriter writer, Int32 originalWriterDepth, Int32 flushThreshold, JsonSerializerOptions options, WriteStack& state)
   at System.Text.Json.JsonSerializer.WriteCore(Utf8JsonWriter writer, Object value, Type type, JsonSerializerOptions options)
   at System.Text.Json.JsonSerializer.WriteValueCore(Utf8JsonWriter writer, Object value, Type type, JsonSerializerOptions options)
   at System.Text.Json.JsonSerializer.Serialize[TValue](Utf8JsonWriter writer, TValue value, JsonSerializerOptions options)
   at NzbDrone.Common.Serializer.STJson.Serialize[TModel](TModel model, Stream outputStream, JsonSerializerOptions options) in D:\a\1\s\src\NzbDrone.Common\Serializer\System.Text.Json\STJson.cs:line 59
   at Radarr.Http.Extensions.NancyJsonSerializer.Serialize[TModel](MediaRange contentType, TModel model, Stream outputStream) in D:\a\1\s\src\Radarr.Http\Extensions\NancyJsonSerializer.cs:line 34
   at Nancy.Responses.JsonResponse`1.<>c__DisplayClass6_0.<GetJsonContents>b__0(Stream stream)
   at Radarr.Http.Extensions.Pipelines.GzipCompressionPipeline.WriteGZipStream(Action`1 innerContent, Stream targetStream) in D:\a\1\s\src\Radarr.Http\Extensions\Pipelines\GZipPipeline.cs:line 76
   at Radarr.Http.Extensions.Pipelines.GzipCompressionPipeline.<>c__DisplayClass6_0.<CompressResponse>b__1(Stream responseStream) in D:\a\1\s\src\Radarr.Http\Extensions\Pipelines\GZipPipeline.cs:line 51
   at Nancy.Owin.NancyMiddleware.RequestComplete(NancyContext context, IDictionary`2 environment, Func`2 performPassThrough, Func`2 next)
   at Nancy.Owin.NancyMiddleware.<>c__DisplayClass2_1.<<UseNancy>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Owin.WebSocketAcceptAdapter.<>c__DisplayClass6_0.<<AdaptWebSockets>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Radarr.Host.Middleware.SignalRMiddleware.<Attach>b__7_0(HttpContext context, Func`1 next) in D:\a\1\s\src\NzbDrone.Host\WebHost\Middleware\SignalRMiddleware.cs:line 51
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application)
```

File: src/Radarr.Api.V3/Indexers/ReleaseResource.cs [hunk 1]
```diff
@@ -51,8 +50,6 @@ public class ReleaseResource : RestResource
         public int? Leechers { get; set; }
         public DownloadProtocol Protocol { get; set; }
 
-        // Sent when queuing an unknown release
-        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
```
Comment (Qstick) [inline src/Radarr.Api.V3/Indexers/ReleaseResource.cs:55]: They added further options for this in net5 (https://github.com/dotnet/runtime/issues/779)

Should we hold off on all the serialization changes until after a 3.0, seems kinda big to change json library for API if its only for minimal gains (based on what you mentioned about the other items being the bulk) until 3.1 (net 5)

Comment (ta264) [reply src/Radarr.Api.V3/Indexers/ReleaseResource.cs]: Might be best to leave this for net 5

**Comment by Qstick** [comment]
Rebased on develop with screenshot changes to prevent build failure

**Comment by Qstick** [comment]
I’m good on this, not sure if we want to try figure out Brian’s issue first or push it in and see if anyone else is effected.

**Pull request merged**